mypy/build.py

```diff
@@ -809,6 +813,44 @@ def remove_cwd_prefix_from_path(fscache: FileSystemCache, p: str) -> str:
     return p
 
 
+USER_SITE_PACKAGES = \
+    'from __future__ import print_function; import site; print(repr(site.getsitepackages()' \
+    ' + [site.getusersitepackages()]))'
+VIRTUALENV_SITE_PACKAGES = \
+    'from distutils.sysconfig import get_python_lib; print(repr([get_python_lib()]))'
```

Comment:
Should we do the `__future__` import here as well for consistency, or not do it all?